_docs/vision.md

```diff
@@ -15,39 +15,39 @@ PostgreSQL?
 
 Welcome to Babelfish. The idea of the project is to close the gap between
 PostgreSQL and other systems by making PostgreSQL speak other protocols such as
-TDS, and maybe a lot more in the future. By implementing "MS SQL"-slang, we make
+TDS, and maybe a lot more in the future. By implementing "Microsoft SQL Server"-slang, we make
```

Comment:
To suggest an alternative sentence (I don't care for the use of the word 'slang' - it has a negative connotation in my mind, like the syntax we've implemented is sloppy):

By implementing support for Microsoft SQL Server syntax, Babelfish makes it easier to move to PostgreSQL.

Reply:
Done; I agree that is better.